application/forms/Command/Object/AddCommentForm.php

```diff
@@ -21,6 +22,15 @@ class AddCommentForm extends CommandForm
 {
     use Auth;
 
+    public function __construct()
+    {
+        $this->on(self::ON_SUCCESS, function () {
+            Notification::success(
+                tp('Added a comment successfully', 'Added comments successfully', count($this->getObjects()))
```

Comment:
* `Added comment successfully` as singular version
* Include count of objects in plural version
* Rephrase plural version to `Added comment successfully to %d hosts` for hosts (for services `Added comment successfully to %d services`)
  It's the same comment on every host/service.

Please adjust all other messages I didn't comment regarding this the same way.